src/core/derived_state.py

```diff
@@ -55,3 +58,4 @@ def update(self, derived_state_dict: Dict) -> None:
         for key, value in derived_state_dict.items():
             if key in self.__dict__.keys():
                 setattr(self, key, value)
+        print(derived_state_dict["r_mo"])
```

Comment:
pls delete all print statements.
If you really need to print something, use logging instead